app/models/container_group.rb

```diff
@@ -37,6 +40,18 @@ def ready_condition_status
     ready_condition.try(:status) || 'None'
   end
 
+  def waiting_containers
+    "#{containers.to_a.count { |container| container.state.eql? 'waiting' }}/#{containers.count}"
```

Comment:
@chessbyte , I simplified the counts and made them numeric. However, in order to make the explore table more informative I used another method to display (as a string) how many containers are running out of the total number of containers in the pod, instead of just the number of running containers (please see the screenshot above).
cc @simon3z 
@dkorn done, thanks.